common/decisions/wc_cen_kal_decisions.txt

```diff
@@ -0,0 +1,370 @@
+		  text = create_thorns_quilboar_req_tt
+		  var:blood_shard_count >= 12
+		}
+	}
+	
+	effect = {
+	custom_tooltip = create_thorns_quilboar_tt
+	random_sub_realm_county = {
+		limit = {
+			OR = {
+				holder = root
+				holder = { is_vassal_or_below_of = root }
+			}
+			title_province = {
+				NOR = {
+					has_building = quilboar_thorns_01
+					has_building = quilboar_thorns_02
+					has_building = quilboar_thorns_03
+				}
+			}
+		}
+		title_province = { save_scope_as = thorn_location }
+	}
+	#Found one
+	if = { limit = { exists = scope:thorn_location } 
+		set_variable = {
+			name = thorn_location
+			value = scope:thorn_location
+		}
+		trigger_event = cen_kal.1055
+	}
+	#No Avalible provinces
+	if = { limit = { NOT = { exists = scope:thorn_location } } trigger_event = cen_kal.1056 }
+	}
+
+	cost = { piety = 50 }
+
+	ai_will_do = { base = 100 }
+}
+
+create_quilboar_empire = {
+	ai_check_interval = 12
+	picture = "gfx/interface/illustrations/decisions/quilboar_empire.dds"
+	desc = create_quilboar_empire_desc
+	title = create_quilboar_empire_name
+	selection_tooltip = create_quilboar_empire_tt
+	confirm_text = create_quilboar_empire_confirm
+	major = yes
+
+	ai_check_interval = 0
+
+	is_shown = {
+		is_landed = yes
+		highest_held_title_tier <= tier_kingdom
+		OR = {
+			faith = faith:agamaggan_worship
+			culture = culture:quilboar
+		}
+		NOT = {  #Can only do it once.
+			is_target_in_global_variable_list = {
+				name = unavailable_unique_decisions
+				target = flag:flag_created_quilboar_empire
+			}
+		}
+		root.primary_title = { is_mercenary_company = no }
+	}
+
+	is_valid = {
+		custom_description = {
+			text = found_empire_decision_two_kingdoms_held
+			any_held_title = {
+				count >= 2
+				tier = tier_kingdom
+			}
+		}
+		title:c_razorfen = {
+			OR = {
+				holder = root
+				custom_tooltip = { text = found_empire_location_owned_or_vassal_tt holder = { is_vassal_or_below_of = root } }
+			}
+		}
+		title:c_downs = {
+			OR = {
+				holder = root
+				custom_tooltip = { text = found_empire_location_owned_or_vassal_tt holder = { is_vassal_or_below_of = root } }
+			}
+		}
+	}
+
+	is_valid_showing_failures_only = {
+		is_available_adult = yes
+		is_imprisoned = no
+		is_independent_ruler = yes
+	}
+
+	cost = {
+		prestige = 2000
+		piety = 1000
+	}
+
+	effect = {
+		save_scope_as = founder
+		create_new_custom_empire_decision_effect = { NAME = quilboar_empire_name } 
+		add_prestige_level = 1
+		root.primary_title = { set_color_from_title = title:d_red_crane }
+		
+		trigger_event = cen_kal.1060
+		#Notify other players.
+		every_player = {
+			limit = {
+				NOT = { this = root }
+				is_within_diplo_range = { CHARACTER = root }
+			}
+			trigger_event = cen_kal.1061
+		}
+		hidden_effect = {
+			add_to_global_variable_list = {
+				name = unavailable_unique_decisions
+				target = flag:flag_created_quilboar_empire
+			}
+			set_global_variable = {
+				name = flag_created_quilboar_empire
+				value = scope:founder
+			}
```

Comment:
```suggestion
```
Never used, delete it

Reply:
It's used in the decision to check that the character hasn't made it before, nor has any other character made that distinct empire.